playbooks/templates/.env.j2

```diff
@@ -1,118 +1,237 @@
-###############################################################################
-# Server specific
-
-# a portal domain name in a form <subdomain>.<domain>.<top-level-domain>
-DOMAIN_NAME={{ webportal_server_config.domain_name }}
-
-# a domain that needs to be set to a domain with tld ie. `siasky.net`
-# this is the portal domain that will be used for generating certificates and nginx routing
-PORTAL_DOMAIN={{ webportal_cluster_config.portal_cluster_domain if (portal_cluster_id is defined) else webportal_server_config.domain_name }}
+# xxxqqq: # xxx: Remove 'skynetlabs_devs_email' from:
+# - .env.j2
+# Review usage (rename) in:
+# - webportals.yml
+# - hosts.ini
+{% set server_domain = webportal_server_config.domain_name %}
+{% set cluster_domain = webportal_common_config.portal_cluster_domain %}
 
-# a domain to set only if you have an alternative domain for a server
-# (if you have multiple servers that make up a portal) ie. `eu-ger-1.siasky.net`
-SERVER_DOMAIN={{ webportal_server_config.domain_name if(portal_cluster_id is defined) else '' }}
+###############################################################################
+# Notes
+#
+# Portal server can be a single, standalone server, i.e. not part of a cluster.
+# Single server domain we will use for examples: single-portal.com
+#
+# Portal server can be part of a cluster.
+# Cluster domains we will use for examples:
+# - Cluster domain: cluster.com
+# - Server domain: server-1.cluster.com
 
-# a string representing name of your portal e.g. `siasky.xyz` or `my skynet
-# portal` (internal use only)
-PORTAL_NAME='{{ webportal_server_config.portal_name }}'
+###############################################################################
+# Server specific
 
-# a list of active portal modules. e.g. 'PORTAL_MODULES=aj'. Possible choices:
-# 'a': accounts, 'j': Jaeger
+# A portal domain name
+# For a single server: single-portal.com
+# For a cluster: server-1.cluster.com
+# xxx: Is this stil used somewhere?
```

Comment:
Did you get an answer on this?

Reply:
As discussed online, will test portals without it and remove it if unused.
Issue #101